CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_shopify_source v0.8.2
+
+## Bug Fixes
+- The `fivetan_utils.union_data` [macro](https://github.com/fivetran/dbt_fivetran_utils/pull/100) has been expanded to handle checking if a source table exists. Previously in the Shopify source package, this check happened outside of the macro and depended on the user having a defined shopify `source`. If the package anticipates a table that you do not have in any schema or database, it will return a **completely** empty table (ie `limit 0`) that will work seamlessly with downstream transformations ([PR #59](https://github.com/fivetran/dbt_shopify_source/pull/59)).
+  - A compilation **warning** when it is completely empty. The compiler warning can be turned off by the end user by setting the global `remove_empty_table_warnings` variable to `True`.
```

Comment:
See my comments from the fivetran_utils package around possibly changing the wording and adding the variable to the message.